### ISSUE TITLE
Automatically invite/add people to Facebook assets

### ISSUE THREAD

**Issue by dahacouk** [drupalsocialinitiative/social_api#1]
I want to be able to automatically add/invite a person to a Facebook Page, Facebook Group and/or Facebook Event by either identifying an individual person or group of people. Obviously, Drupal would need to know their Facebook ID. Currently I have to manually go to each Facebook Page, Facebook Group and/or Facebook Event and add people individually.

So, one scenario is that a person registers on a website and provides their Facebook details and then can be added/invited automatically to Facebook assets.

**Comment by RoySegall** [comment]
This will  be for Drupal 8? If so the infrastructure would need be very much plugin based, we would need some UX/UI for the forms of the invention etc. etc.

**Comment by dahacouk** [comment]
Yes, I think we've all agreed that we target Drupal 8. In #2 @drupalnomad is going to describe that process of plugins. Perhaps you could collaborate and bring some references to a document in https://goo.gl/YDvjxL ?
